lib/mix/tasks/baz.gen.migration.ex

```diff
@@ -31,7 +31,9 @@ defmodule Mix.Tasks.Baz.Gen.Migration do
         |> Enum.map(fn source_path ->
           EEx.eval_file(source_path,
             module_prefix: app_module,
-            table_and_index_prefix: Baz.Migrations.table_and_index_prefix()
+            table_and_index_prefix: Baz.Migrations.table_and_index_prefix(),
```

Comment:
Would this be :scissors: in the final PR?

Reply:
Definitely.